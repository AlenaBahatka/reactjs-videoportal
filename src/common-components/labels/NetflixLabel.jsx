import React from 'react';

function NetflixLabel() {
	return <span>netflixroulette</span>;
}

export default NetflixLabel;
