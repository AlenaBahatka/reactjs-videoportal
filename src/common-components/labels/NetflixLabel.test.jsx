import React from 'react';
import { create } from 'react-test-renderer';
import NetflixLabel from './NetflixLabel';

describe('NetflixLabel', () => {
    it('Netflix label is correct', () => {
        const netflixLabelComponent = create(<NetflixLabel/>);
        expect(netflixLabelComponent.toJSON()).toMatchSnapshot();
    })
});


