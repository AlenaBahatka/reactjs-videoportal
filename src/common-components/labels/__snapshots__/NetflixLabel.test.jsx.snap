// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetflixLabel Netflix label is correct 1`] = `
<span>
  netflixroulette
</span>
`;
